src/DssSpell.sol

```diff
@@ -73,189 +51,105 @@ contract DssSpellAction is DssAction {
 
-        // Set price feed for RWA-00x
-        DssExecLib.setContract(DssExecLib.spotter(), collateral.ilk, "pip", pip);
+        uint256 removed = add(univ2EthUsdtLine, univ2DaiUsdtLine);
 
-        // Init RWA-00x in Vat
-        Initializable(vat).init(collateral.ilk);
-        // Init RWA-00x in Jug
-        Initializable(DssExecLib.jug()).init(collateral.ilk);
+        uint256 Line = VatAbstract(MCD_VAT).Line();
+        VatAbstract(MCD_VAT).file("Line", sub(Line, removed));
 
-        // Allow RWA-00x Join to modify Vat registry
-        DssExecLib.authorize(vat, collateral.MCD_JOIN);
+        // -----------  Increase UNIV2DAUUSDC-A Maximum Debt Ceiling -----------
+        // https://vote.makerdao.com/polling/QmRAuMND?network=mainnet#poll-detail
 
-        // Set ilk/global DC
-        DssExecLib.increaseIlkDebtCeiling(collateral.ilk, collateral.CEIL, true);
+        DssExecLib.setIlkAutoLineParameters("UNIV2DAIUSDC-A", 250 * MILLION, 10 * MILLION, 8 hours); // 50 million to 250 million.
 
-        // Set stability fee
-        DssExecLib.setIlkStabilityFee(collateral.ilk, collateral.RATE, false);
+        // ----------- Housekeeping -----------
 
-        // Set collateralization ratio
-        DssExecLib.setIlkLiquidationRatio(collateral.ilk, collateral.MAT);
+        // Update RWA tokens symbols in ilk registry
+        IlkRegistryAbstract ILK_REGISTRY = IlkRegistryAbstract(DssExecLib.reg());
 
-        // Poke the spotter to pull in a price
-        DssExecLib.updateCollateralPrice(collateral.ilk);
+        ILK_REGISTRY.file("RWA001-A", "symbol", "RWA001");
+        ILK_REGISTRY.file("RWA002-A", "symbol", "RWA002");
+        ILK_REGISTRY.file("RWA003-A", "symbol", "RWA003");
+        ILK_REGISTRY.file("RWA004-A", "symbol", "RWA004");
+        ILK_REGISTRY.file("RWA005-A", "symbol", "RWA005");
+        ILK_REGISTRY.file("RWA006-A", "symbol", "RWA006");
 
-        // Set up the urn
-        Hopeable(collateral.URN).hope(collateral.OPERATOR);
+        // Update early RWA tokens names in ilk registry
+        ILK_REGISTRY.file("RWA001-A", "name", "RWA001-A: 6s Capital");
+        ILK_REGISTRY.file("RWA002-A", "name", "RWA002-A: Centrifuge: New Silver");
 
-        // Add RWA-00x contract to the changelog
-        DssExecLib.setChangelogAddress(collateral.gemID, collateral.GEM);
-        DssExecLib.setChangelogAddress(collateral.pipID, pip);
-        DssExecLib.setChangelogAddress(collateral.joinID, collateral.MCD_JOIN);
-        DssExecLib.setChangelogAddress(collateral.urnID, collateral.URN);
-        DssExecLib.setChangelogAddress(
-            collateral.inputConduitID, collateral.INPUT_CONDUIT
-        );
-        DssExecLib.setChangelogAddress(
-            collateral.outputConduitID, collateral.OUTPUT_CONDUIT
-        );
+        // Add vote delegate factory to changelog
+        DssExecLib.setChangelogAddress("VOTE_DELEGATE_PROXY_FACTORY", VOTE_DELEGATE_PROXY_FACTORY);
```

Comment:
It's too late now, but I wish we called this simply: `VOTE_DELEGATE_FACTORY`.  I didn't think of it when we were doing `kovan`.  This is fine.